SYCL/DeprecatedFeatures/histogram_256_slm_spec.cpp

```diff
@@ -0,0 +1,207 @@
+//==--------------- histogram_256_slm.cpp  - DPC++ ESIMD on-device test ----==//
```

Comment:
- The test path should have esimd somewhere (DeprecatedFeatures/esimd?)
- As I see, SYCL/ESIMD/mandelbrot/mandelbrot_spec.cpp was fixed in this PR to use 2020 spec constants, can the same be done for other tests (e.g. this one)?

Reply:
> The test path should have esimd somewhere (DeprecatedFeatures/esimd?)

Fixed

> can the same be done for other tests (e.g. this one)?

I'm not sure. Some tests have a SYCL 2020 version (like this one), and others don't, and I don't see any pattern here.